app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] The like belongs to the author, kindly consider implementing this so the relations are setup correctly
e.g ``belongs_to :author, class_name: 'User'``